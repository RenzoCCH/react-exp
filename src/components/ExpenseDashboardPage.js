import React from 'react';
import ExpenseList from './ExpenseList.js';
import ExpenseListFilters from './ExpenseListFilters.js';
import ExpensesSummary  from './ExpensesSummary.js';


const ExpenseDashboardPage = ()=>(
  <div>
    <ExpenseListFilters/>
    <ExpensesSummary/>
    <ExpenseList/>
  </div>
);

export default ExpenseDashboardPage;