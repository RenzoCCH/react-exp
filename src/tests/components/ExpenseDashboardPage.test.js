import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';

test('should test expense dashboard page', () => {
  const wrapper = shallow(<ExpenseDashboardPage/>);
  expect(wrapper).toMatchSnapshot();
});
