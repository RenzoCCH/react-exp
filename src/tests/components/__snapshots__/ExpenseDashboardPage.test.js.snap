// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test expense dashboard page 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <ExpensesSummary />
  <Connect(ExpenseList) />
</div>
`;
